app.cr

```diff
@@ -9,13 +9,20 @@ def headers
   headers
 end
 
-def crystal_repos
+def crystal_repos(sort)
   client = HTTP::Client.new("api.github.com", 443, true)
-  response = client.get("/search/repositories?q=language:crystal&per_page=100", headers)
+  response = client.get("/search/repositories?q=language:crystal&per_page=100&sort=#{sort}", headers)
```

Comment:
I'm a bit concerned about this, since it will make a github API request every time we change the sort order. What do you think of having some sort of local cache? It could be on the server or on the client.

We could also go with this for now, and when/if we have a real problem with, fix it then. Ideas?